WebRoot/messages/ZmMsg.properties

```diff
@@ -4478,20 +4479,31 @@ QUERY_IS_unreplied = not replied to by me
 
 # Two step authentication setting dialog
 twoStepAuthSetup = Set up Two-step authentication
+twoStepAuthSetupLoginPage = Two-Factor Authentication
```

Comment:
These might not exactly be in alphabetical order. Maybe you wanna reorder these since babel reorders alphabetically?

Reply:
Some strings are up and down , so i have kept similar strings together that are used acrosss two strings